### DISCUSSION TITLE
Save Last State of Element View (Status and Search)

### DISCUSSION THREAD

**Discussion by RitterKnightCreative** [craftcms/cms#9026] [Ideas]
### Description
At the top of every "index" view, there's the Status dropdown which lets you select what shows up (ie All, Live, Draft, etc.)

There's also a search box letting you find specific elements.

![Screen Shot 2021-04-20 at 9 08 29 PM](https://user-images.githubusercontent.com/3707869/115486588-93f4a580-a21c-11eb-907b-9ad673fa0f2e.png)

It would be nicer for usability if these views were "sticky" and set to that value until changed.  

For example, If I go into an entry then immediately hit the back button (or make a change and save) I'm returned to the All view which is annoying if you're on Drafts or some other view that doesn't have the entry you were just working on.  

Also, when if you've searched for a specific thing,  Craft will also clear out the current search if there is one (in other words treat it like a new page) which is less than ideal on a page with a lot of entries or assets.

I'm not sure if this change was made recently in response to something else recently (maybe the drafts) but I was just testing an older version of Craft (eg 3.5) and this behavior works as expected (though the search still clears).

### Additional info

- Craft version: 3.6.11.2
- PHP version: 7.4

**Comment by brandonkelly** [comment]
> I'm not sure if this change was made recently in response to something else recently (maybe the drafts) but I was just testing an older version of Craft (eg 3.5) and this behavior works as expected (though the search still clears).

The only thing that has ever been remembered between page loads is the selected source.

**Comment by brandonkelly** [comment]
Related, we should probably store these things in the URL as well, to make the current view state easier to share with others.
